monkey/monkey_island/cc/services/ransomware_report.py

```diff
@@ -0,0 +1,8 @@
+from monkey_island.cc.services.reporting.report import ReportService
+
+
+def get_exploitation_details():
+    scanned = ReportService.get_scanned()
+    exploited = ReportService.get_exploited()
```

Comment:
What exactly do we need for statistics? There's no need to send all exploited and especially scanned nodes. Maybe the count is enough?

Reply:
Won't we need the details for the table (we need the machine, how it was exploited, and ransomware simulation details)?

Reply:
I think the front end does very little (if any) report processing. So we could do all of the processing on the back end and send just the information that will be displayed.

```python
{"numScanned":  len(scanned), "numExploited": len(exploited), "exploits" {"wmi": 7, "smb": 13 ... }}
```

Reply:
So the ransomware report won't show the exploit that gave access successful to a particular machine?

Reply:
Done
![image](https://user-images.githubusercontent.com/44770317/125250037-7282f180-e313-11eb-8821-041ea82e538c.png)